resolwe/flow/views/entity.py

```diff
@@ -138,6 +151,21 @@ def add_data(self, request, pk=None):
 
         return resp
 
+    @list_route(methods=['post'])
+    def move_to_collection(self, request, *args, **kwargs):
+        """Move samples from source to destination collection."""
+        entity_ids = self.get_ids('entity_ids', request.data)
+        source_collection_id = self.get_id('source_collection', request.data)
```

Comment:
`src_collection_id`, `dst_collection_id`, `src_collection`, `dst_collection`. For variables, you can keep the long form in the request.